aspnetcore/mvc/views/tag-helpers/builtin/resources/AnchorTagHelper.md

```diff
@@ -62,11 +54,15 @@ you would get generated the html
 
 This is because there was no route found that matched a controller that had a method named ```Detail``` with one string parameter titled ```name```.
 
-## asp-route
+- - -
+
+### asp-route
 
 ```asp-route``` provides a way to create a URL that links directly to a named route. Using routing attributes, you can name your routes as shown in the Evaluations method above of the ```SpeakerController```.  ```Name = "speakerevals"``` tells the AnchorTagHelper to generate a route directly to that controller method using the URL ```/Speaker/Evaluations```. If ```asp-controller``` or ```asp-action``` is specified in addition to ```asp-route``` then the route generated will not be what you expect.  ```asp-route``` should not be used with either of the attributes ```asp-controller``` or ```asp-action``` to avoid a route conflict.
 
-## asp-all-route-data
+- - -
+
+### asp-all-route-data
 
 ```asp-all-route-data``` allows you to create on your current .net context (that is, the running c# associated with your razor page) a dictionary of key value pairs where the key is the parameter name and the value is the value associated with that key. As the example below shows, you can create an inline dictionary on your razor page and then create the ```AnchorTagHelper``` right after that. You could of course pass in the dictionary with your model or as your model. That avoids having extra c# code on your razor page and gives you better control of the data passed to your view.
```

Comment:
> ```asp-all-route-data``` allows you to create on your current .net context (that is, the running c# associated with your razor page) a dictionary of key value pairs where the key is the parameter name and the value is the value associated with that key.

* ".net" to ".NET"
* "c#" to "C#"
* "razor page" to "Razor view" ("Razor page" has special meaning; Razor is a proper noun)
* It's confusing to inject that much parenthetical information (i.e., "that is ...") into the sentence. I think if you explain what the context is first, that the sentence will become easier to digest.

> You could of course pass in

@tdykstra will be fine with that :grin: (inside joke), but I try to avoid splitting verbs when it isn't necessary. How about just "Of course, you could pass in ..."

> ... data passed to your view.

Since the data will appear in the view, I wonder if this would go "data passed into your view."